core/components/Themes.php

```diff
@@ -0,0 +1,188 @@
+<?php
```

Comment:
In general, yes i agree we could added a themes component to luya which will be the main channel to communicate whether a theme exists or not, and which one is active. etc. So maybe we could keep this component, but improve it and make it the main channel to communicate between controller, views and application.